src/applications/vaos/components/EligibilityCheckMessage.jsx

```diff
@@ -7,11 +7,19 @@ export default function EligibilityCheckMessage({ eligibility }) {
       <div aria-atomic="true" aria-live="assertive">
         <AlertBox
           status="warning"
-          headline="Sorry, we couldn't find a recent appointment"
+          headline="We couldn’t find a recent appointment at this location"
         >
-          In order to request an appointment at this facility for the chosen
-          type of care, you need to have been seen within the past{' '}
-          {eligibility.requestPastVisitValue} months.
+          <p>
+            You need to have visited this facility within the past{' '}
+            {eligibility.requestPastVisitValue} months to request an appointment
```

Comment:
Not sure if it's possible for `requestPastVisit` value to equal 1, but if so, "months" could be conditionally plural.  Same for line 19 below.

Reply:
In theory, we're getting a number back and it could be anything, but I believe SM limits it to 12 or 24.